lib/dw/visualization.base.js

```diff
@@ -57,6 +57,15 @@ _.extend(base, {
         return this;
     },
 
+    target(target) {
+        if (!arguments.length) {
+            return this.__target;
```

Comment:
would be good to check if the target has been set here and otherwise throw an exception